UIKit/Tasks/Model.swift

```diff
@@ -0,0 +1,24 @@
+// Model.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Класс модель
+struct Model {
```

Comment:
```suggestion
struct Model {
```
Model не пишем никогда